lib/ansible/plugins/lookup/cloudformation_parse.py

```diff
@@ -0,0 +1,52 @@
+# (c) 2018, Steven Miller
+# (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: cloudformation_parse
+    author: Steven Miller (@sjmiller609)
+    version_added: "2.6"
+    short_description: Parse cloudformation files
+    description:
+        - Allows users to parse json or yaml cloudformation files into a list of json objects
+    options:
+      _terms:
+        description: File name or list of filenames for the cloudformation template(s) to load
+        required: True
+"""
+
+EXAMPLES = """
+- debug: msg="{{ lookup('cloudformation_parse','myfile.yml') }}"
+"""
+
+RETURN = """
+  _list:
+    description:
+      - The parsed cloudformation file(s)
+    type: list
+"""
+
+# This module allows us to parse cfn including special tags
+# The module is supported by awslabs
+# see 'load' function here
+# https://github.com/awslabs/aws-cfn-template-flip/blob/master/cfn_flip/__init__.py
+from cfn_flip import load as cfn_load
```

Comment:
Please add a try/expect around this import like in https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/consul_kv.py#L108-L123